autoortho/aoconfig.py

```diff
@@ -147,6 +192,32 @@ def ui_loop(self):
                 default_value=maptype, key='maptype_override',
                 metadata={'section':self.cfg.autoortho})],
             [sg.HorizontalSeparator(pad=5)],
+            [
+                sg.Text('Cache size in GB'),
+                sg.Slider(
+                    range=(10,100,5),
```

Comment:
Can't we allow a bigger range here? It could be useful for testing to clear the whole cache and someone might want to keep 1TB (why not?).
```suggestion
                    range=(0,1000,5),
```

Reply:
Oh these somehow uneducated inexplicably cache-averse users... Alright ;)